src/sql/workbench/test/electron-browser/api/extHostModelViewDialog.test.ts

```diff
@@ -105,10 +105,6 @@ suite('ExtHostModelViewDialog Tests', () => {
 	});
 
 	test('Button clicks are forwarded to the correct button', () => {
-		// Set up the proxy to record button handles
```

Comment:
this is not a reliable way to record the handle of buttons, this method will be called every time a property is set, I added a position property for button earlier, and that caused this method being called twice per button, as a result handles[0] and handles[1] are both the same button. I have exposed the getHandle method to reliably get handle.